### ISSUE TITLE
Add entries for contain:style and other property values

### ISSUE THREAD

**Issue by foolip** [mdn/browser-compat-data#14036]
https://developer.mozilla.org/en-US/docs/Web/CSS/contain#browser_compatibility

Different kinds of containment are coming to different browsers at different times. In the there's currently [4 types of containment](https://drafts.csswg.org/css-contain/#propdef-contain), although that doesn't map cleanly to 4 property values.

@andruud which property value do you think would make sense to record separately here in BCD and thus on MDN? Or should it not be in terms of the property values, but rather the kinds of containment in the abstract?

**Comment by andruud** [comment]
These are the things that might be interesting to include somehow. Not sure if all of them warrant "full" entries.

- `contain:style` is not implemented in all browsers.
- `contain:inline-size` was added recently in [css-contain-3](https://drafts.csswg.org/css-contain-3/#contain-property).
- `contain:strict` now has an old and a new interpretation per a [recent-ish resolution](https://github.com/w3c/csswg-drafts/issues/6287#issuecomment-862525199).

But note that this is just information I've picked up passively, I've never actually looked for discrepancies.
